docker-compose/docker-compose.yaml

```diff
@@ -9,4 +9,27 @@ services:
     environment:
       - TZ=Europe/Zurich
     command: ["--verbose", "--https-port", "8443", "--global-response-templating"]
+    networks:
+      - cdr_client_net
 
+  mock-oauth2-server:
+    image: ghcr.io/navikt/mock-oauth2-server:2.1.9
+    environment:
+      - TZ=Europe/Zurich
+      - LOG_LEVEL=DEBUG
+      - JSON_CONFIG_PATH=/app/config.json
+      - SERVER_PORT=8443
+    ports:
+      - "8444:8443"
+    volumes:
+      - ./mockOAuth2Server/config.json:/app/config.json
+      - ./mockOAuth2Server/mockoauth2server.p12:/app/mockoauth2server.p12
+    networks:
+      - cdr_client_net
+
+networks:
+  cdr_client_net:
+    name: cdr_client_net
+    ipam:
+      config:
+        - subnet: 10.112.0.0/16
```

Comment:
Please bump sub-net to `10.113.0.0/16` (10.112 is already taken by the `cdr-app-mgr` compose network.